src/flat-button.jsx

```diff
@@ -145,6 +146,7 @@ const FlatButton = React.createClass({
     const buttonHoverColor = hoverColor || defaultHoverColor;
     const buttonRippleColor = rippleColor || defaultRippleColor;
     const hovered = (this.state.hovered || this.state.isKeyboardFocused) && !disabled;
+    const buttonBackgroundColor = backgroundColor || contextProps.buttonColor;
```

Comment:
`contextProps` is no longer used. You probably need to change this to:

```js
const buttonBackgroundColor = backgroundColor || buttonColor;
```

Also, I was thinking that it would be nice to add the `backgroundColor` property to the `flatButton` key inside `theme-manager` too. What do you think? That way, we can check in this order of precedence:

`backgroundColor` prop > `backgroundColor` theme value > `buttonColor`

cc @oliviertassinari